Doc/c-api/init.rst

```diff
@@ -1239,12 +1239,24 @@ All of the following functions must be called after :c:func:`Py_Initialize`.
    The global interpreter lock need not be held, but may be held if it is
    necessary to serialize calls to this function.
 
+   .. audit-event:: cpython.PyThreadState_New "" c.PyThreadState_New
+
+      Raise an auditing event ``cpython.PyThreadState_New`` with no arguments.
+      The event will be raised from the creating thread, not the new thread.
```

Comment:
This wording might be confusing in some situations as isn't the creating thread the new thread in some circumstances such as when C/C++ has created a thread which is then calling back into the Python runtime?   Just stating "will be raised from the thread creating the new PyThreadState which may not be the new thread." would be more accurate in that case.